rpmbuild/copr-rpmbuild.spec

```diff
@@ -47,6 +47,12 @@ BuildRequires: %{python_pfx}-munch
 BuildRequires: %{python}-requests
 BuildRequires: %{python_pfx}-jinja2
 BuildRequires: %{python_pfx}-simplejson
+%if 0%{?fedora}
+BuildRequires: %{python}-retry
+%endif
+%if 0%{?rhel}
+BuildRequires: %{python}-retrying
```

Comment:
Can we have a dummy test-case that the lib works?

Reply:
The lib doesn't work (our irc chat). But I found the exact same package in epel https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2023-46680e2ac7